pkgs/development/ocaml-modules/bls12-381/default.nix

```diff
@@ -1,21 +1,17 @@
-{ lib, buildDunePackage, fetchFromGitLab, ff-sig, zarith
-, zarith_stubs_js, integers_stubs_js, integers, hex
-, ff-pbt, bisect_ppx, alcotest
-}:
+{ lib, buildDunePackage, fetchFromGitLab, ocaml
```

Comment:
The `ocaml` argument is not needed, is it?

Reply:
It's the platform thing again :see_no_evil: fixed